src/Utilities/src/MoistThermodynamics.jl

```diff
@@ -261,7 +267,7 @@ on temperature `T` allows analytic integration of the Clausius-Clapeyron
 relation to obtain the saturation vapor pressure `p_vs` as a function of
 the triple point pressure `press_triple`.
 """
-function sat_vapor_press_generic(T, LH_0, cp_diff)
+function sat_vapor_press(T, LH_0, cp_diff, phase::Generic)
```

Comment:
rather than use a `Generic` singleton, we can just make this the 3-argument form of the function.